contents/docs/libraries/js/index.mdx

```diff
@@ -458,6 +458,20 @@ The `/decide` endpoint can be disabled by setting `advanced_disable_decide = tru
 
+## AutocaptureConfig
+
+`autocapture` config can take an object providing further config to autocapture's behavior.
+
+| Attribute                                                                                                                                                                                 | Description                                                                                                                    |
+| ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------ |
+| `url_allowlist`<br/><br/>**Type:** Array of String or Regexp<br/>**Default:** `undefined`                                                                                                          | List of URLs to allow autocapture on, can be strings to match or regexes e.g. ['https://example.com', 'test.com/.*']. NB an empty list means no URLs are allowed for capture.                                                                                                 |
+| `dom_event_allowlist`<br/><br/>**Type:** `'click' or 'change' or 'submit'`<br/>**Default:** `undefined`                                                                                                          | List of DOM events to allow autocapture on.          NB an empty list means no events are allowed for capture.                                                                                            |
+| `element_allowlist`<br/><br/>**Type:** `'a' or 'button' or 'form' or 'input' or 'select' or 'textarea' or 'label'`<br/>**Default:** `undefined`                                                                                                          | List of DOM elements to allow autocapture on.          NB an empty list means no elements are allowed for capture.                                                                                            |
+| `css_selector_allowlist`<br/><br/>**Type:** `String[]`<br/>**Default:** `undefined`                                                                                                          | List of CSS selectors to allow autocapture on.          NB an empty list means no css selectors are allowed for capture.                                                                                            |
+| `element_attribute_ignorelist`<br/><br/>**Type:** `String[]`<br/>**Default:** `undefined`                                                                                                          | List of element attributes that autocapture will not capture.          NB an empty list means no attributes are ignored.                                                                                     |
```

Comment:
Clarification needed here.
```suggestion
| `element_attribute_ignorelist`<br/><br/>**Type:** Array of Strings <br/>**Default:** `undefined`                                                                                                          | An array of element attributes that autocapture will not capture. An empty array means all events are ignored, `undefined` means elements with any attribute are captured.                                                                                     |
```

Reply:
```suggestion
| `element_attribute_ignorelist`<br/><br/>**Type:** Array of Strings <br/>**Default:** `undefined` |  An array of element attributes that autocapture will not capture. Both an empty array and `undefined` mean any of the attributes from the element are captured. |
```

Reply:
This one is slightly different since its an `ignore` value and not an `allow` value. Edited slightly based on your suggestion but obvs feel free to edit further 😊

Reply:
Ahh, that's good. Thanks!